src/components/PublicRoadmap.astro

```diff
@@ -0,0 +1,60 @@
+---
+export const prerender = false;
+
+interface Task {
+  id: string;
+  name: string;
+  description: string;
+  status: string;
+}
+
+const PUBLIC_ROADMAP_ENDPOINT = "https://depth-108-knife-781-buy-129-bat.hook.membrane.io/";
```

Comment:
It's not a _huge_ deal, but it may just be better to store this in an environment variable. We don't necessary want folks hitting this a ton.

Reply:
Fair point. I added an environment variable in Vercel, but I am having trouble finding a happy path to access it in an Astro project. I've poked around [Vercel docs on Astro](https://vercel.com/docs/frameworks/astro#astro-on-vercel) and [Astro docs on environment variables](https://docs.astro.build/en/guides/environment-variables/), among other places.